src/style.css

```diff
@@ -23,7 +23,7 @@ body {
   font-family: Pxlvetica, Arial, sans-serif;
   font-size: x-large;
   margin: 0;
-  height: 100vh;
+  height: 100dvh;
```

Comment:
did not know about this unit

Reply:
tbh idek if its doing anything... im prob gonna change it all to grid anyway